css/components/_figure.scss

```diff
@@ -13,5 +13,10 @@ figure {
     border-top: 1px solid $gray--mid;
     padding: 2px 0;
     font-size: 0.8rem;
+
+    &.thick {
```

Comment:
Really? I don't particularly like how it looks. Do you like the big padding or the bold font or both?